src/coreclr/vm/jitinterface.cpp

```diff
@@ -13941,20 +13942,34 @@ BOOL LoadDynamicInfoEntry(Module *currentModule,
 
     case ENCODE_INDIRECT_PINVOKE_TARGET:
         {
-            MethodDesc *pMethod = ZapSig::DecodeMethod(currentModule, pInfoModule, pBlob);
+            if (mayUsePrecompiledNDirectMethods)
```

Comment:
I do not think you need to suppress this one. Resolving `ENCODE_INDIRECT_PINVOKE_TARGET` should not be loading any unmanaged .dlls.

Reply:
(The suppression for `ENCODE_PINVOKE_TARGET` should stay.)

Reply:
Thanks